layouts/trackers/single.html

```diff
@@ -93,6 +95,10 @@ <h2>{{ T "tracker-single-transmitted-data-heading" }}</h2>
                                     -->
                                     {{ if or (strings.HasPrefix .reasoning "https://") (strings.HasPrefix .reasoning "http://") }}
                                         <a href="{{ .reasoning }}"><img src="/svg/information.svg" class="inline-icon icon-information" alt="{{ T "tracker-single-transmitted-data-table-reasoning-link-desc" }}" title="{{ T "tracker-single-transmitted-data-table-reasoning-link-desc" }}"></a>
```

Comment:
In fact, I'm not sure about this, either. At the very least, I think we should have different icons (and `title` texts) for links and our own documentation. These are clearly very different and that should be discernible at a glance imo. Also, #17…

Reply:
At least in the long, I would also really like to come up with a better option for displaying these altogether. But we shouldn't waste time on that now.

Reply:
> At least in the long run, I would also really like to come up with a better option for displaying these altogether. But we shouldn't waste time on that now.

https://github.com/tweaselORG/tracker-wiki/issues/18

Reply:
I agree, I also don't like the current "solution", but I didn't have a better idea so here we are.